### DISCUSSION TITLE
I clone the project and build the demo, some files are missing

### DISCUSSION THREAD

**Discussion by angelen10** [LeoNatan/LNPopupController#586] [General] [answered]
<img width="783" alt="image" src="https://github.com/user-attachments/assets/30d23fbd-65cf-45c5-8f4b-1b7436d16e1f">
<img width="683" alt="image" src="https://github.com/user-attachments/assets/5142cc5d-19b2-485a-bffb-8b0163a2cb32">

**Comment by LeoNatan** [comment, marked as answer]
Hello,
The demo project uses git submodules. Make sure to clone those too. 
Cheers

**Reply by LeoNatan** [reply]
You can use the `git submodule update --init` command. I guess I should add that information to the readme.
